build.gradle

```diff
@@ -13,11 +9,22 @@ ext {
     krystal_version = '3.15-SNAPSHOT'
```

Comment:
its 4 now right?

Reply:
I've not yet published 4.
Will do one we get all the performance improvements etc checked in